app/Http/Controllers/Random.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class Random extends Controller
```

Comment:
コントローラーには`Controller`をつけましょう😊
つけなければいけない決まりではないですが、コントローラーにはつけておくのが一般的でわかりやすいです。
クラス名、ファイル名の修正をお願いします。
```suggestion
class RandomController extends Controller
```